### PR TITLE
COL-919, my pinned first in listing of Whiteboards > Add asset

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/COL-919

John approves of this PR. But will Paul?
 
image

### CODE REVIEW

File: public/app/whiteboards/reuse/reuse.html [hunk 1]
```diff
@@ -12,9 +12,45 @@ <h2 class="modal-title">Add asset(s)</h2>
...
+
+          <div data-ng-if="!isSearch">
+            <h3 data-ng-if="!isSearch">Pinned Assets</h3>
+            <ul>
+              <li class="list-inline col-xs-6 col-sm-4 col-md-3"
+                data-ng-repeat="asset in assets | filter:{pinned_by_me: true} | unique:'id'">
+
+                <div class="col-list-item-container">
+                  <a data-ng-click="selectAsset(asset)">
+                    <div class="col-list-item-tile">
```
Comment (paulkerschen) [inline public/app/whiteboards/reuse/reuse.html]: @johncrossman Could the newly created `shared/assetThumbnail.html` be subbed in here (and under "All Assets" below)? Free badges!

Comment (johncrossman) [inline public/app/whiteboards/reuse/reuse.html]: You had me at "free", @paulkerschen.  Let me check...

Comment (johncrossman) [inline public/app/whiteboards/reuse/reuse.html]: Commit amended. PR is getting better all the time ("...couldn’t get no worse").

File: public/app/assetlibrary/item/item.html [hunk 2]
```diff
@@ -21,24 +21,33 @@
...
     <div class="col-xs-{{ (2 * upperRightButtonCount) + 1 }} col-sm-{{ (2 * upperRightButtonCount) + 1 }} text-right">
-      <a ui-sref="assetlibrarylist.item.edit({assetId: asset.id})" class="btn btn-default" data-ng-if="canManageAsset()">
+      <a ui-sref="assetlibrarylist.item.edit({assetId: asset.id})" class="btn btn-default"
+        data-ng-if="canManageAsset()">
         <i class="fa fa-pencil"></i>
         <span>Edit details</span>
       </a>
-      <a class="assetlibrary-item-pin btn btn-default" data-ng-click="pin()" class="btn btn-default">
+      <button id="pin-asset" class="assetlibrary-item-pin btn btn-default"
+        data-ng-click="pin()">
```
Comment (johncrossman) [inline public/app/assetlibrary/item/item.html]: **Note:** I took this opportunity to make 'pin' a button and a static `id` for all.

**Comment by johncrossman** [comment]
Sample UI:
image

**Comment by paulkerschen** [comment]
This PR gets badges and stripes.

splh11

**Pull request merged**